tests/process-modeler/process-modeler.page.ts

```diff
@@ -48,9 +48,23 @@ export class ProcessModelerPage {
 
     const pageURL = page.url();
     const processDefinitionID = pageURL.split('/processes/').pop();
+
     this.processDefinitionID = processDefinitionID;
     this.processName = processName;
     this.processDescription = description;
+
+    await this.waitForHydration();
```

Comment:
Isn't this only needed for the shortcut tests? After initial load, the site is always hydrated on navigation so we should only wait for this if needed, to not delay any further.

Reply:
Thats true -> made the waiting optional and made a new test-fixture for the shortcuts